deploy/demo/aadpodidentity-sp.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: demo-aad1-sp
+type: Opaque
+data:
+  ClientSecret: CLIENT_SECRET
+---
+apiVersion: "aadpodidentity.k8s.io/v1"
+kind: AzureIdentity
+metadata:
+  name: demo-aad1
+spec:
+  type: 1
+  TenantID: TENANT_ID
+  ClientID: CLIENT_ID
+  ClientPassword: {"Name":"demo-aad1-sp","Namespace":"default"}
```

Comment:
First of all there shoule be a Service Principal and you have ClientId/Secret of it, in aadpodidentity-sp.yaml CLIENT_SECRET represents sp secret, TENANT_ID represents tenant id of sp, CLIENT_ID represents ClientId of sp.

Follow the tutorial https://github.com/Azure/aad-pod-identity/blob/master/docs/tutorial/README.md, in section 3.1 deploy the aadpodidentity-sp.yaml  instead of aadpodidentity.yaml.

Reply:
My request here is that you add these steps above to the README. It’s fine if we do this in a future PR.

Reply:
I am not good at write a good README, I prefer a seprate PR to improve the document.

Reply:
Sure, no problem.